src/index.html

```diff
@@ -10,5 +10,31 @@
   <body>
     <div id="main"></div>
     <script src="./js/index.js"></script>
+    <ul>
+      <button background-color: transparent; style="width:200px;height:96px" ;>
```

Comment:
`background-color: transparent;` should be included in the `style=""` attribute.
However it is not recommended to use inline styles since it will be hard to override later on.

Reply:
Thank you for your comments. So I can start a CSS file, right?

Reply:
No, you don't need to. It's not a part of requirement for lesson 1. So for lesson 1, you are required to implement the HTML structure only, but not including any CSS.

I think Cody just wants to point it out that the inline style should be avoided since it's not easy to maintain and extended.